### DISCUSSION TITLE
rockchip-6.1 kernel

### DISCUSSION THREAD

**Discussion by black-device** [radxa-repo/rbuild#54] [Ideas]
hello!

due to the release of rockchip-6.1 kernel,
Do you think it is possible to start testing for CM3 builds under debian and ubuntu with this kernel?

It would be nice to see the advantages that are supposed to exist in it, and it would also be desirable to see a compilation preparing things for Ubuntu numbat, which will be the next LTS.

Thank you!

**Comment by RadxaYuntian** [comment]
This kernel currently supports RK3588 only. Ubuntu is also very hard to support since Rockchip SDK is Debian only.
